index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Document</title>
+    <script type="text/javascript" src="dist/envspec.js"></script>
+     <link rel="stylesheet" href="dist/styles.css">
+  </head>
+
+  <body>
+    <script type="text/javascript" src="dist/demoPage.js"></script>
```

Comment:
1. This should be moved down, in the line above the closing tag of body (above `</body>`).
2. This file is now ignored, since it is inside the `dist` directory, move it to `demo/main.js` to track it in Git